src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.video.unit.spec.jsx

```diff
@@ -920,9 +920,9 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
       ).to.be.ok;
     });
 
-    it.skip('should verify Video Connect at home calendar ics file format', async () => {
+    it('should verify Video Connect at home calendar ics file format', async () => {
       const appointment = getVideoAppointmentMock();
-      const startDate = moment.utc().add(3, 'days');
+      const startDate = moment(getTimezoneTestDate()).add(3, 'days');
```

Comment:
Modified to use 'getTimeezoneTestDate' which generates a date that when adjusted for a timezone will result in a new date on the previous day. In my case, the generated date/time string is 12:30am CST with the timezone information stripped. This string when used by the app will be adjusted to America/Denver MDT (based on the location id for Cheyenne VA medical center) which is 1 hour behind.